src/main/java/org/terasology/rendering/nui/layers/hud/InventoryHud.java

```diff
@@ -33,10 +35,19 @@
 
     private UICrosshair crosshair;
 
+    // Set "true" to use the rotating style quickslot; set "false" to get the default style quickslot
+    @LayoutConfig
+    private boolean rotateItems = true;
```

Comment:
If "false" gets us the default style quickslot, then the default value of `rotateItems` should be `false`
```suggestion
    // Set "true" to use the rotating style quickslot; set "false" to get the default style quickslot
    @LayoutConfig
    private boolean rotateItems = false;
```

Reply:
Yes indeed. I left it as such to make tests at first.